docs/enable-apps.md

```diff
@@ -22,12 +25,13 @@ If following the setup guide, stop when you reach the section **Running your Rea
 Then, create a new React Native project from the template:
 
 ```shell
-npx react-native@latest init AwesomeProject
+npx react-native@latest init AwesomeProject --skip-install
```

Comment:
Not sure if this is necessary here. Why have you added it @lunaleaps ?

Reply:
This was from this comment: https://github.com/reactwg/react-native-new-architecture/pull/156#discussion_r1505943602

Also realized I forgot to copy the rest of the nit, updated!

Reply:
It just save a bit of time, avoiding the to run pod install  twice